.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,21 @@
+---
+name: 📃 Documentation Bug
+about: You want to report something that is wrong or missing from the documentation.
```

Comment:
Suggest: "You want to suggest improvements or report something that is wrong or missing from the documentation"